flict/flictlib/translator.py

```diff
@@ -61,23 +69,19 @@ def parse():
                         dest='translation_file',
                         help='files with translations, default is ' + DEFAULT_TRANSLATIONS_FILE,
                         default=DEFAULT_TRANSLATIONS_FILE)
-    parser.add_argument('-p', '--package-file',
-                        type=str,
-                        dest='package_file',
-                        help='file with package definition')
+#    parser.add_argument('-p', '--package-file',
```

Comment:
Why disable them? Could you kindly provide some insights. Even if they're used in the current setup it's worth making the removal a separate commit

Reply:
I actually got an opposite impression. Please look at the comment above. Actual usage is currently limited to importing single function from that file, however there was also argparse, as it would have been intended to be used separately, from command line. However, `parse()` was never called, in case the file was executed from command line. As i didn't know the history, and more importantly the reason or idea behind that - instead of removing them i thought it might be more beneficial to keep them commented out - after enabling proper parsing from command line, an user could be surprised reading help, that graph or packaging file could be provided to the script, while in fact they weren't used, nor the code for graph creation was there. So, long story short - i thought this would prevent confusing user, prevent misleading.